documentation/NUnit1038.md

```diff
@@ -0,0 +1,82 @@
+# NUnit1038
+
+## The type of the attribute values doesn't match the parameter type
+
+| Topic    | Value
+| :--      | :--
+| Id       | NUnit1038
+| Severity | Error
+| Enabled  | True
+| Category | Structure
+| Code     | [RangeUsageAnalyzer](https://github.com/nunit/nunit.analyzers/blob/master/src/nunit.analyzers/RangeUsage/RangeUsageAnalyzer.cs)
+
+## Description
+
+Ensure that the attribute and parameter types match.
+
+## Motivation
+
+The `Range` attribute is used to specify a range of values for a parameter in a test method.
+The type of the range values must match the type of the parameter.
+If NUnit cannot match the types, it will throw an exception at runtime.
+NUnit rules are that `int` can be assigned to any integral type and `double`.
```

Comment:
I'm not a native English speaker, but I found the end of the sentence to sound a little odd (I cannot explain why)
```suggestion
NUnit rules are that an `int` can be assigned to any integral type or to a `double`.
```

Reply:
From the tests it also seems like int can be assigned to decimal?